test/e2e/azure_privatecluster.go

```diff
@@ -458,3 +456,22 @@ func getAPIVersionFromUserAgent(userAgent string) string {
 	splits := strings.Split(userAgent, "/")
 	return splits[len(splits)-1]
 }
+
+func getClientIDforMSI(resourceID string) string {
```

Comment:
Can we add a doc comment for this function?

Reply:
added